ktmidi/src/commonMain/kotlin/dev/atsushieno/ktmidi/MidiPlayer.kt

```diff
@@ -252,7 +252,7 @@ enum class PlayerState {
     PAUSED,
 }
 
-interface SeekProcessor<TMessage> {
+fun interface SeekProcessor<TMessage> {
```

Comment:
At this state I don't plan to make this interface as a SAM (it would be a breaking change if I add anything further, unless it becomes a @ JvmDefault method), unless there is any strong reason to make it SAM.

Reply:
Sure, I'll remove it! I just added it everywhere I could haha